src/PrestaShopBundle/Resources/config/services/core/grid/grid_position_definition_factory.yml

```diff
@@ -48,3 +48,12 @@ services:
       $positionField: 'position'
       $parentIdField: 'id_product'
       $firstPosition: 1
+
+  prestashop.core.grid.product.position_definition:
+    class: 'PrestaShop\PrestaShop\Core\Grid\Position\PositionDefinition'
+    arguments:
+      $table: 'category_product'
+      $idField: 'id_product'
+      $positionField: 'position'
+      $parentIdField: 'id_category'
+      $firstPosition: 1
```

Comment:
So we can define the first position here, then why do we need to increment it in the template, seems like we should always rely on the actual position even when it's 0-based since it's configurable anyway